when autoapplying spoof blocks notify user what their hidden host is (same as /sethost or +rx)
update numeric reply to reflect new conf format, blocks instead of lines

diff -r b03d03214a42 include/numeric.h
--- a/include/numeric.h	Thu Mar 18 09:57:00 2010 +0100
+++ b/include/numeric.h	Thu Mar 18 10:59:37 2010 +0100
@@ -314,7 +314,7 @@
 /*      RPL_NOUSERS          395        Dalnet/EFnet/IRCnet */
 #define RPL_HOSTHIDDEN       396	/* UMODE +x completed succesfuly */
 #define RPL_STATSSLINE       398	/* QuakeNet extension -froo */
-#define RPL_USINGSLINE       399	/* QuakeNet extension -froo */
+/*      RPL_USINGSLINE       399	   QuakeNet extension -froo */
 
 /*
  * Errors are in the range from 400-599 currently and are grouped by what
diff -r b03d03214a42 ircd/s_conf.c
--- a/ircd/s_conf.c	Thu Mar 18 09:57:00 2010 +0100
+++ b/ircd/s_conf.c	Thu Mar 18 10:59:37 2010 +0100
@@ -1270,8 +1270,9 @@
         continue;
 
       ircd_strncpy(cli_user(cptr)->host, hostonly, HOSTLEN);
-      log_write(LS_USER, L_INFO, LOG_NOSNOTICE, "S-Line (%s@%s) by (%#R)",
+      log_write(LS_USER, L_INFO, LOG_NOSNOTICE, "Spoofhost (%s@%s) by (%#R)",
           cli_user(cptr)->username, hostonly, cptr);
+      SetSetHost(cptr);
       return 1;
     }
   }
diff -r b03d03214a42 ircd/s_err.c
--- a/ircd/s_err.c	Thu Mar 18 09:57:00 2010 +0100
+++ b/ircd/s_err.c	Thu Mar 18 10:59:37 2010 +0100
@@ -830,7 +830,7 @@
 /* 398 */
   { RPL_STATSSLINE, "%d %s %s %s %s", "398" },
 /* 399 */
-  { RPL_USINGSLINE, ":Using S-line privilege", "399" },
+  { 0 },
 /* 400 */
   { 0 },
 /* 401 */
diff -r b03d03214a42 ircd/s_user.c
--- a/ircd/s_user.c	Thu Mar 18 09:57:00 2010 +0100
+++ b/ircd/s_user.c	Thu Mar 18 10:59:37 2010 +0100
@@ -373,12 +373,11 @@
 
     if (feature_bool(FEAT_AUTOINVISIBLE))
       SetInvisible(sptr);
-    
+
+    /* apply auto sethost if needed */
     if(feature_bool(FEAT_SETHOST_AUTO)) {
-      if (conf_check_slines(sptr)) {
-        send_reply(sptr, RPL_USINGSLINE);
-        SetSetHost(sptr);
-      }
+      if (conf_check_slines(sptr))
+        sendrawto_one(sptr, "NOTICE AUTH :*** Using Spoof block privilege");
     }
 
     SetUser(sptr);
@@ -456,6 +455,9 @@
    */
   if (HasHiddenHost(sptr))
     hide_hostmask(sptr, FLAG_HIDDENHOST);
+  /* client has sethost, inform them of their hidden host */
+  if (HasSetHost(sptr))
+    send_reply(sptr, RPL_HOSTHIDDEN, cli_user(sptr)->host);
   if (IsInvisible(sptr))
     ++UserStats.inv_clients;
   if (IsOper(sptr))
